### PR TITLE
[single node perf] Recalibrate and improve regression perf test

### PR DESCRIPTION
**Pull request by igor-aptos**
Recalibrate for RG change
Update limits to be based on min_ratio / max_ratio of many runs update module working set to 100

## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] Breaking change
- [ ] Performance improvement
- [ ] Refactoring
- [ ] Dependency update
- [ ] Documentation update
- [ ] Tests

## Which Components or Systems Does This Change Impact?
- [ ] Validator Node
- [ ] Full Node (API, Indexer, etc.)
- [ ] Move/Aptos Virtual Machine
- [ ] Aptos Framework
- [ ] Aptos CLI/SDK
- [ ] Developer Infrastructure
- [ ] Move Compiler
- [ ] Other (specify)

## Checklist
- [ ] I have read and followed the [CONTRIBUTING](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md) doc
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I identified and added all stakeholders and component owners affected by this change as reviewers
- [ ] I tested both happy and unhappy path of the functionality
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: testsuite/single_node_performance.py [hunk 1]
```diff
@@ -766,6 +811,12 @@ def print_table(
 if errors:
     print("Errors: ")
     print("\n".join(errors))
+    print("""If you expect your PR to change the performance, you need to recalibrate the values.
```
Comment (georgemitenkov) [inline testsuite/single_node_performance.py:814]: Thanks!

**Review by georgemitenkov — APPROVED** [review]

File: testsuite/single_node_performance.py (hunk 1 above)
Comment (msmouse) [reply testsuite/single_node_performance.py:814]: Thanks!

File: testsuite/single_node_performance.py [hunk 2]
```diff
@@ -143,58 +143,63 @@ class RunGroupConfig:
...
+CALIBRATED_COUNT_INDEX = -4
+CALIBRATED_MIN_RATIO_INDEX = -3
+CALIBRATED_MAX_RATIO_INDEX = -2
 CALIBRATION_SEPARATOR = "	"
 
-# transaction_type	module_working_set_size	executor_type	min_ratio	max_ratio	median
+# transaction_type	module_working_set_size	executor_type	count	min_ratio	max_ratio	median
 # (or if from log:
 #    transaction_type module_working_set_size executor_type  block_size expected_tps tps
 # )
```
Comment (msmouse) [inline testsuite/single_node_performance.py:154]: remove?

File: testsuite/single_node_performance.py [hunk 3]
```diff
@@ -143,58 +143,63 @@ class RunGroupConfig:
 
 # 0-indexed
 CALIBRATED_TPS_INDEX = -1
+CALIBRATED_COUNT_INDEX = -4
+CALIBRATED_MIN_RATIO_INDEX = -3
+CALIBRATED_MAX_RATIO_INDEX = -2
 CALIBRATION_SEPARATOR = "	"
 
-# transaction_type	module_working_set_size	executor_type	min_ratio	max_ratio	median
+# transaction_type	module_working_set_size	executor_type	count	min_ratio	max_ratio	median
```
Comment (msmouse) [inline testsuite/single_node_performance.py:151]: where does "count" come from originally again? (I read the query and it seems the binary prints it?)

Comment (msmouse) [reply testsuite/single_node_performance.py:151]: nvm, it's `count(as="count")`

File: .github/workflows/workflow-run-execution-performance.yaml [hunk 4]
```diff
@@ -17,6 +17,11 @@ on:
         default: false
         type: boolean
         description: Run complete version of the tests
+      IS_MAINNET_RUN:
```
Comment (sionescu) [inline .github/workflows/workflow-run-execution-performance.yaml:20]: Is this used anywhere ?

File: .github/workflows/workflow-run-execution-performance.yaml [hunk 5]
```diff
@@ -36,15 +41,25 @@ on:
...
+        - LAND_BLOCKING
+        - CONTINUOUS
+        - MAINNET
+        - MAINNET_LARGE_DB
+        type: choice
+        description: Which set of tests to run. MAINNET/MAINNET_LARGE_DB are for performance validation of mainnet nodes.
       IGNORE_TARGET_DETERMINATION:
         required: false
-        default: false
+        default: true
```
Comment (sionescu) [inline .github/workflows/workflow-run-execution-performance.yaml:62]: Are you sure it's a good idea to ignore by default ?

**Review by sionescu — APPROVED** [review]

File: .github/workflows/workflow-run-execution-performance.yaml (hunk 5 above)
Comment (igor-aptos) [reply .github/workflows/workflow-run-execution-performance.yaml:62]: this is only for jobs triggered from the UI, right?

if we are triggering from the UI, we really shouldn't be skipping if there are no changes?

File: .github/workflows/workflow-run-execution-performance.yaml (hunk 4 above)
Comment (igor-aptos) [reply .github/workflows/workflow-run-execution-performance.yaml:20]: stale

File: .github/workflows/workflow-run-execution-performance.yaml (hunk 5 above)
Comment (sionescu) [reply .github/workflows/workflow-run-execution-performance.yaml:62]: Yes, you're right.

**Pull request merged**